pkg/controllers/metrics/controller.go

```diff
@@ -75,7 +75,7 @@ func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 func (c *Controller) Register(_ context.Context, m manager.Manager) error {
 	return controllerruntime.
 		NewControllerManagedBy(m).
-		Named(controllerName).
+		Named(strings.ToLower(controllerName)).
```

Comment:
consider just making the variable lower, rather than lowering it in both places.

Reply:
Done